data/crac-creation/crac-creator-cim/src/test/java/com/farao_community/farao/data/crac_creation/creator/cim/parameters/JsonVoltageCnecsCreationParametersTest.java

```diff
@@ -55,7 +55,7 @@ void roundTripOnVoltageMonitoringParameters() throws IOException {
         String exportedString = os.toString();
 
         InputStream inputStream = getClass().getResourceAsStream("/parameters/voltage-cnecs-creation-parameters-for-round-trip.json");
-        assertEquals(new String(inputStream.readAllBytes()), exportedString);
+        assertEquals(new String(inputStream.readAllBytes()).replaceAll("\r", ""), exportedString.replaceAll("\r", ""));
```

Comment:
I'm running and developping on a windows WSL2 machine and checkouted the code on a windows host. The unit test are failing because git converts the end of line into a windows style  ie: Line Feed + Carriage return. By using replaceAll on the line feed caracters the unit tests are now passing also on windows hosts